less/plugins/adapt-contrib-media/media.less

```diff
@@ -1,12 +1,51 @@
-.media-component {
-  .media-transcript {
-    &:hover {
-      color: @button-text-color-hover;
-      background-color: @button-color-hover;
+.media {
+  &__transcript-container {
+    margin-top: @item-margin;
+  }
+
+  &__transcript-btn {
+    padding: @item-padding;
+    background-color: @item-color;
+    color: @item-color-inverted;
+    border-radius: 5px;
+
+    .no-touch &:hover {
+      background-color: @item-color-hover;
+      color: @item-color-inverted-hover;
+      .transition(background-color @duration ease-in, color @duration ease-in;);
     }
   }
-  .media-transcript-button-container {
-  	padding-left: 0px;
-  	padding-right: 0px;
+
+  &__transcript-body-inline {
+    margin-top: @item-margin;
+    padding: @item-padding;
   }
-}
\ No newline at end of file
+
+  // --------------------------------------------------
```

Comment:
Should these be deleted?

Reply:
Beginnings of customisable mejs media player, would recommend it remains so work can continue.